process/mock/feeAccumulatorStub.go

```diff
@@ -25,7 +33,7 @@ func (f *FeeAccumulatorStub) GetAccumulatedFees() *big.Int {
 }
 
 // ProcessTransactionFee -
-func (f *FeeAccumulatorStub) ProcessTransactionFee(cost *big.Int) {
+func (f *FeeAccumulatorStub) ProcessTransactionFee(cost *big.Int, txHash []byte) {
 	if f.ProcessTransactionFeeCalled != nil {
 		f.ProcessTransactionFeeCalled(cost)
```

Comment:
Apply the same signature to the mock method ProcessTransactionFeeCalled

Reply:
done